extension/chrome/settings/setup.ts

```diff
@@ -97,14 +97,14 @@ Catch.try(async () => {
       if (!GoogleAuth.hasReadScope(storage.google_token_scopes || [])) {
         $('.auth_denied_warning').css('display', 'block');
       }
-      if (typeof storage.addresses === 'undefined') {
+      if (typeof storage.sendAs === 'undefined') {
         if (GoogleAuth.hasReadScope(storage.google_token_scopes || [])) {
```

Comment:
This `if` check no longer makes sense (sorry, I overlooked this last time).

We needed to check if we had read permission because we used to guess aliases based on sent emails, and read permission was needed to examine sent emails.

We now use proper api for this, so this check is not needed.